pkg/compose/create.go

```diff
@@ -838,13 +837,7 @@ func (s *composeService) buildContainerVolumes(
 	var mounts []mount.Mount
 	var binds []string
 
-	img := api.GetImageNameOrDefault(service, p.Name)
```

Comment:
moved to `buildContainerMountOptions` as this is only used when we manage anonymous volumes inheritance (allows to write test without a useless API call/mock)